cmd/minishift/cmd/start.go

```diff
@@ -353,6 +353,7 @@ func initClusterUpFlags() {
 	clusterUpFlagSet.Int(serverLogLevel, 0, "Log level for the OpenShift server.")
 	clusterUpFlagSet.StringSliceVarP(&openShiftEnv, openshiftEnv, "e", []string{}, "Specify key-value pairs of environment variables to set on the OpenShift container.")
 	clusterUpFlagSet.Bool(metrics, false, "Install metrics (experimental)")
+	clusterUpFlagSet.Bool(logging, false, "Install logging (experimental)")
```

Comment:
So what type of logging are we talking about here? This is the OpenShift integrated logging, right? (similar to metrics)? This is not logging in terms of CLI logging and stands in no relation to glog or the global log level, right?